src/pages/Pool/v2.tsx

```diff
@@ -178,9 +178,11 @@ export default function Pool() {
                 <ResponsiveButtonPrimary
                   id="find-pool-button"
                   as={Link}
-                  padding="6px 8px"
-                  borderRadius="12px"
                   to="/pool/v2/find"
+                  style={{
```

Comment:
the idea is to pass those props to `ResponsiveButtonPrimary` which is a `styled(RebassButton)`. I think we should be able to not use `style` here.

https://github.com/Uniswap/uniswap-interface/blob/1556ff26b28b8c2bff2229d9e9b1e901fda13d59/src/components/Button/index.tsx#L16